src/main/java/seedu/address/model/car/Car.java

```diff
@@ -0,0 +1,101 @@
+package seedu.address.model.car;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Objects;
+
+import seedu.address.commons.util.ToStringBuilder;
+
+/**
+ * Represents a Car in MATER.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Car {
+
+    private final Vrn vrn; // Vehicle Registration Number (Vrn).
```

Comment:
According to [the feature spec](https://docs.google.com/document/d/1SLxukdZaTKmlAaD_-vpuevYnVJrAFfK4N_fvJbP3zhc/edit) the edit command can edit these fields, i think we will need to update the feature spec

Reply:
NVM the edit command should create a new instance of the class, not change the field